src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -214,33 +222,37 @@ def generate_MCP_1_default_config(self, filename):
     }
 
     # Generate SAS Test Harnesses dump records for multiple iterations   
-    sas_test_harness_0_iteration_0_dump_records = {
+    dump_records_iteration_0_sas_test_harness_0 = {
         'cbsdRecords': cbsd_fad_records_iteration_0_sas_test_harness_0
     }
-    sas_test_harness_0_iteration_1_dump_records = {
+    dump_records_iteration_1_sas_test_harness_0 = {
         'cbsdRecords': cbsd_fad_records_iteration_1_sas_test_harness_0
     }
-    sas_test_harness_1_iteration_0_dump_records = {
+    dump_records_iteration_0_sas_test_harness_1 = {
         'cbsdRecords': cbsd_fad_records_iteration_0_sas_test_harness_1
     }
-    sas_test_harness_1_iteration_1_dump_records = {
+    dump_records_iteration_1_sas_test_harness_1 = {
         'cbsdRecords': cbsd_fad_records_iteration_1_sas_test_harness_1
     }
 
     # Create the actual config.
     iteration0_config = {
-        'cbsdRequests': [cbsd_records_iteration_0_domain_proxy_0,cbsd_records_iteration_0_domain_proxy_1],
+        'cbsdRequestsWithDomainProxys': [cbsd_records_iteration_0_domain_proxy_0,cbsd_records_iteration_0_domain_proxy_1],
+        'registrationRequests': [device_7],
```

Comment:
Typically we take them as an array because that is easy to do with a DP. In the case of an individual CBSD, it is probably better to put the information for each CBSD into its own dict so that we can also specify the cert/key pair to be used.

Reply:
Updated   cbsd record as suggested in the comment